packages/mongo-nav/src/mongo-select/Trigger.tsx

```diff
@@ -62,7 +58,7 @@ const border = css`
 
 interface OrganizationTriggerProps {
   children?: React.ReactNode;
-  current: CurrentOrganizationInterface;
+  current: string;
```

Comment:
Not sure I understand this change. Why did we make this type more broad?

Reply:
we were accepting an object, now just a string

Reply:
we don't need all of the other information from the CurrentOrganizationInterface object, simpler to just pass the name of the currentOrg

Reply:
Shouldn't this be a string union of the possible values?

Reply:
It's whatever a user names their org, so could be anything

Reply:
We should rename this prop because as is it's confusing what it means, ditto for project.

Reply:
Should we call the prop `placeholder` as @DesignerDave suggested?